app/helpers/serialize_intervention_helper.rb

```diff
@@ -0,0 +1,25 @@
+module SerializeInterventionHelper
+  def serialize_intervention(intervention)
+    {
+      id: intervention.id,
+      name: intervention.name,
+      comment: intervention.comment,
+      goal: intervention.goal,
+      start_date: intervention.start_date.strftime('%B %e, %Y'),
+      end_date: intervention.end_date.try(:strftime, '%B %e, %Y'),
+      educator_email: intervention.educator.try(:email),
+      progress_notes: intervention.progress_notes.order(created_at: :asc).map do |progress_note|
```

Comment:
Makes sense! Good point that educators could add / edit interventions that change the ordering.